tutorials/vue/img.md

```diff
@@ -0,0 +1,139 @@
+To use the `img` tag in vue, you must use the `v-bind:src` directive or `:src` for short.
+Be sure to enclose the image address in single quotes on top of the double quotes.
+
+```javascript
+  const app = new Vue({
+    template: `
+      <div style="width: 50%">
+       <img :src="'../../assets/logo.png'" />
+      </div>
+    `,
+  });
+  app.$mount("#content");
+```
+
+<div id="content"></div>
+<script src="https://cdn.jsdelivr.net/npm/vue@2.6.12"></script>
+<script>
+  const app = new Vue({
+    template: `
+      <div style="width: 50%">
+       <img :src="'../../assets/logo.png'" />
+      </div>
+    `,
+  });
+  app.$mount("#content");
+</script>
+
+## Computed src
+
+You can also pass a computed property to `:src`, but make sure that it returns the image file location.
```

Comment:
What does "image file location" mean here?

Reply:
I'll change it to "path to the image file"